Assets/HoloToolkit-Examples/Utilities/Readme/README_SolverSystem.md

```diff
@@ -0,0 +1,41 @@
+# [Solver System]()
```

Comment:
Will Assets\HoloToolkit\UX components use a Solver? If yes, then why are they not part of the core toolkit? Under UX? Or a new folder may be?

Reply:
I would hate for a core component to depend on examples.